identity/webapp/src/frontend/MyAccount/ChangePassword.test.tsx

```diff
@@ -242,11 +242,11 @@ describe('ChangePassword', () => {
     });
   });
 
-  xdescribe('shows an error after submission', () => {
+  describe('shows an error after submission', () => {
     it('when the current password is incorrect', async () => {
       server.use(
         rest.put('/account/api/users/me/password', (req, res, ctx) => {
-          return res(ctx.status(401));
+          return res.once(ctx.status(401));
```

Comment:
Any idea why this is better?

Reply:
I suspect it was something else that fixed things but the intention is that the handler is for that test only (doesn't have to be overridden)